library/Faktory/Connection.hs

```diff
@@ -9,9 +9,11 @@ module Faktory.Connection
 import Faktory.Prelude
 
 import Control.Applicative ((<|>))
+import qualified Control.Exception as E
 import Data.Maybe (fromMaybe)
 import Data.Void
 import Network.Connection
+import qualified Network.Socket as S
```

Comment:
This you also don't need, except for `Network.Socket.connect` which otherwise conflicts with the `connect` defined in this module. I might turn `Network.Socket` into an unscoped import to match the style in the rest of this module.